src/cljs/witan/ui/ajax.cljs

```diff
@@ -41,14 +41,16 @@
 
 (defn get-headers
   [response]
-  (ajax.protocols/-get-all-headers response))
+  (as-> (ajax.protocols/-get-all-headers response) h
+    (reduce-kv (fn [a k v] (assoc a (clojure.string/lower-case k) v)) {} h)
+    (update h "vary" #(when % (clojure.string/lower-case %)))))
```

Comment:
Is the `when` needed?
Doesn't the "vary" have to be in `[]`?

Reply:
- Without the `when`, if "vary" is not present then it'll be attempting to call `clojure.string/lower-case` on `null` which will throw an exception.
- Only need `[]` for `update-in`

Reply:
ah okay didn't notice you'd switched to update rather than update-in